tox.ini

```
@@ -0,0 +1,25 @@
+[tox]
+minversion = 3.7.0
+skip_missing_interpreters = true
+envlist =
+    py{37,38,39}
+
+[testenv]
+passenv = DD_SOURCE_*  DD_DESTINATION_*
```

```suggestion
passenv = DD_SOURCE_*  DD_DESTINATION_* PYTEST_*
```
to allow for `PYTEST_ADOPTS="--ddtrace"`